Base/QTCore/qSlicerSceneBundleReader.cxx

```diff
@@ -30,6 +30,7 @@
 #include <ctkUtils.h>
 
 // MRML includes
+#include <vtkCollection.h>
```

Comment:
```suggestion
```

Add below in the `// VTK includes` section

Reply:
thank you, fixed